src/components/backend-ai-splash.ts

```diff
@@ -114,7 +113,13 @@ export default class BackendAISplash extends LitElement {
     // language=HTML
     return html`
       <backend-ai-dialog id="splash-panel" fixed backdrop blockscrolling persistent narrowLayout hideActions>
-        <div class="splash-header" slot="title">
+        <div class="splash-header" slot="title"
+          style="background-image:url(${
+            localStorage.getItem('backendaiwebui.settings.isDarkMode')
+              ? 'manifest/backend.ai-text-bgdark.svg'
+              : 'manifest/backend.ai-text.svg'
+          });"
+        >
```

Comment:
Please check `About` modal text:
<img width="345" alt="image" src="https://github.com/lablup/backend.ai-webui/assets/621215/ecc3a628-48cd-4ebc-8208-6988bb7763e6">

Please change the color of `Release Note License` and `open-source software` for dark mode.